feat: add Standard Notes Standalone Infrastructure setup instructions (#35)
standardnotes/docs: docs/self-hosting/getting-started.md

## Syncing Server

We offer free and fast sync for unlimited devices to both free and paid users. If you are interested in hosting your own syncing server, you are welcome to do so.

## Infrastructure Synopsis

Syncing Server infrastructure consists of a few different microservices that are responsible for different sets of functionality.

> In order to make your life easier and not having to set up each of them separately we prepared a [docker-compose setup with accompanying script](./docker.md) that will handle it for you.

Reviewer: ```suggestion
In order to make your life easier and not having to set up each of them separately we prepared a [docker-compose setup with accompanying script](./docker.md) that will automate the process for you.
```
